core-plugin/testSrc/com/google/gct/idea/debugger/CloudDebugProcessTest.java

```diff
@@ -72,7 +74,7 @@ public void testRegisterAdditionalActions_close() {
 
         assertEquals(3, leftToolbar.getChildrenCount());
         assertEquals(action0, leftToolbar.getChildActionsOrStubs()[0]);
-        assertEquals(action2, leftToolbar.getChildActionsOrStubs()[1]);
+        assertEquals(action3, leftToolbar.getChildActionsOrStubs()[1]);
```

Comment:
Indeed. So, besides a unit test for CloudDebugHelpAction, is there anything else left in this change?

Do I have to notify you to accept this pull?